### PR TITLE
Document adding members to orgs with the 'influx' CLI

### LINKED ISSUE #492: Document adding members to orgs with the 'influx' CLI

### PR DESCRIPTION
**Pull request by pierwill**
Closes #492.

### CODE REVIEW

File: content/v2.0/organizations/members/add-member.md [hunk 1]
```diff
@@ -7,25 +7,33 @@ menu:
...
+<!-- 1. Click the **Settings** tab in the navigation bar. -->
 
-    {{< nav-icon "settings" >}}
+<!--     {{< nav-icon "settings" >}} -->
 
-2. Click on the name of an organization, then select the **Members** tab.
+<!-- 2. Click on the name of an organization, then select the **Members** tab. -->
 
-_Complete content coming soon_
+<!-- _Complete content coming soon_ -->
```
Comment (sanderson) [inline content/v2.0/organizations/members/add-member.md]: No need to comment out each line individually. Just comment out the enter section in a single comment.

File: content/v2.0/organizations/members/add-member.md [hunk 2]
```diff
@@ -7,25 +7,33 @@ menu:
...
 
 ## Add a member to an organization using the influx CLI
 
-Use the [`influx org members add` command](/v2.0/reference/cli/influx/org/members/add)
-to add a user to an organization. Adding a user requires the following:
+Use the [`influx org members add` command](/v2.0/reference/cli/influx/org/members/add) to add a user to an organization.
+You will need the ID of the user you are trying to add.
 
-_Complete content coming soon_
+1. Get a user's ID by running `influx org members list -n <org-name>`.
```
Comment (sanderson) [inline content/v2.0/organizations/members/add-member.md]: 1. This command lists existing members of the org. Users are created outside of an organization, then added as a member of an organization. You need to use the `influx user find` command to list users with their IDs, then use the user ID to add them as a member of an org.
2. Command examples should be in code blocks, not inline code.

File: content/v2.0/organizations/members/add-member.md [hunk 3]
```diff
@@ -7,25 +7,33 @@ menu:
...
+
+2. Add the user with this command:
+
+    ```sh
+    influx org members add -n <org-name> -o <user-ID>
+    ```
+
+{{% cloud-msg %}}
+Adding users is currently unavailable in InfluxDB Cloud 2.
```
Comment (sanderson) [inline content/v2.0/organizations/members/add-member.md]: - Move this to the top of the doc, just under the initial introduction.
- Adding ~users~ **members** is currently...
  "Users" and "members" are two different things. Users exist outside of organizations. A user can be added as a member of multiple organizations.

File: content/v2.0/organizations/members/add-member.md [hunk 4]
```diff
@@ -7,25 +7,33 @@ menu:
...
 
-_Complete content coming soon_
+<!-- _Complete content coming soon_ -->
 
 ## Add a member to an organization using the influx CLI
 
-Use the [`influx org members add` command](/v2.0/reference/cli/influx/org/members/add)
-to add a user to an organization. Adding a user requires the following:
+Use the [`influx org members add` command](/v2.0/reference/cli/influx/org/members/add) to add a user to an organization.
+You will need the ID of the user you are trying to add.
```
Comment (kelseiv) [inline content/v2.0/organizations/members/add-member.md]: I'd prob drop line 26 and 27.

1. Find a user's ID by running..... 
2. Add the member to an organization by running....
For more info on this command, see [`influx org...].

File: content/v2.0/organizations/members/add-member.md [hunk 5]
```diff
@@ -7,25 +7,33 @@ menu:
...
-draft: true
 ---
 
-Use the InfluxDB user interface (UI) or the `influx` command line interface (CLI)
-to add a member to an organization.
+Use the `influx` command line interface (CLI) to add a member to an organization.
 
-## Add a member to an organization in the InfluxDB UI
+{{% cloud-msg %}}
+Adding members is currently unavailable in InfluxDB Cloud 2.
```
Comment (kelseiv) [inline content/v2.0/organizations/members/add-member.md]: InfluxDB Cloud 2.0 (to be consistent w other docs)

Comment (sanderson) [reply content/v2.0/organizations/members/add-member.md]: Or just the `{{< cloud-name >}}` shortcode.

File: content/v2.0/organizations/members/add-member.md [hunk 6]
```diff
@@ -7,25 +7,36 @@ menu:
...
 
-Use the [`influx org members add` command](/v2.0/reference/cli/influx/org/members/add)
-to add a user to an organization. Adding a user requires the following:
+1. Get a list of users and their IDs by running the following:
+
+```sh
+influx user find
+```
+
+2. Add the desired user to an organization with this command:
```
Comment (sanderson) [inline content/v2.0/organizations/members/add-member.md]: - Add the desired user **as a member of** an organization...

I think we need to be really clear that there's a distinction between a user and a member.

File: content/v2.0/organizations/members/add-member.md [hunk 7]
```diff
@@ -7,25 +7,36 @@ menu:
...
 
-Use the [`influx org members add` command](/v2.0/reference/cli/influx/org/members/add)
-to add a user to an organization. Adding a user requires the following:
+1. Get a list of users and their IDs by running the following:
+
+```sh
+influx user find
+```
+
+2. Add the desired user as a member of an organization with this command:
```
Comment (kelseiv) [inline content/v2.0/organizations/members/add-member.md]: To add a user to an organization, run the following command:

**Pull request merged**